libs/HTML/QuickForm2/Controller.php

```diff
@@ -70,12 +70,12 @@ class HTML_QuickForm2_Controller implements IteratorAggregate
    /**
     * Key in $_REQUEST array that contains the ID of the Controller
     */
-    const KEY_ID = '_qfc_id';
+    public const KEY_ID = '_qfc_id';
```

Comment:
We shouldn't apply changes in `/libs`. The folder is excluded from phpcs check anyway

Reply:
Reverted all changes in the `/libs` folder.